src/test/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitterTest.kt

```diff
@@ -66,15 +66,17 @@ internal class HMPPSDomainEventsEmitterTest {
   @Captor
   private val telemetryAttributesCaptor: ArgumentCaptor<Map<String, String>>? = null
 
+  private val hmppsQueueService = mock<HmppsQueueService>()
+  private var hmppsEventSnsClient = mock<AmazonSNSAsync>()
+
   @BeforeEach
   fun setUp() {
+    hmppsEventSnsClient = mock()
```

Comment:
Need to refresh the client to ensure interactions checked in each test are specific to that test